client/src/components/App.js

```diff
@@ -22,20 +22,20 @@ class App extends Component {
     ])
       .then(res => Promise.all(res.map(dataset => dataset.json())))
       .then(data => {
-        console.log(data, "DATA");
         this.setState({ users: data[0], matches: data[1], teams: data[2] });
+        console.log(this.state.matches);
       })
       .catch(err => console.log(err));
   }
 
   render() {
-    if (this.state.users.length === 0) {
+    if (this.state.teams.length !== 32) {
```

Comment:
will this always be 32?

Reply:
For the world cup yes, but for other tournaments no. I will change it.